docs/cloud-soar/cloud-soar-apis.md

```diff
@@ -0,0 +1,22 @@
+---
+id: cloud-soar-apis
+title: Cloud SOAR APIs
+sidebar_label: Cloud SOAR APIs
+description: Learn how to access Cloud SOAR APIs and API documentation.
+---
+
+The Cloud SOAR APIs allow you to manage incidents, triage, and other Cloud SOAR features.
+
+{@import ../reuse/api-intro.md}
+
+| Deployment | Documentation URL                                         |
+|:-----------|:----------------------------------------------------------|
+| AU         | https://api.au.sumologic.com/docs/csoar/  |
+| CA         | https://api.ca.sumologic.com/docs/csoar/  |
+| DE         | https://api.de.sumologic.com/docs/csoar/  |
+| EU         | https://api.eu.sumologic.com/docs/csoar/  |
+| IN         | https://api.in.sumologic.com/docs/csoar/  |
+| JP         | https://api.jp.sumologic.com/docs/csoar/  |
+| US1        | https://api.sumologic.com/docs/csoar/     |
+| US2        | https://api.us2.sumologic.com/docs/csoar/ |
```

Comment:
Will you copy this over to [Sumo APIs](https://help.sumologic.com/docs/api/)?

Reply:
Done.